install_config/install/docker_registry.adoc

```diff
@@ -498,7 +550,7 @@ not the route name and port. See `oc get imagestreams` for details.
 +
 [NOTE]
 ====
-In this example, `test` in the URL `<host>/test/busybox` refers to the project
+In this example, in the url `<host>/test/busybox`, `test` refers to the project
```

Comment:
Caps done, but then because I didn't like the original (`test` was too far from the `refers to...`) but you preferred it, I rewrote it again. LMKWYT (let me know what you think).